<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TeacherWin</class>
 <widget class="QMainWindow" name="TeacherWin">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>779</width>
    <height>572</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>TeacherWin</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_17">
      <item>
       <spacer name="horizontalSpacer_13">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>478</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_19">
        <item>
         <widget class="QLabel" name="label_13">
          <property name="text">
           <string>姓名：</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_14">
          <property name="text">
           <string>***</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_10">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>58</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QLabel" name="label_15">
          <property name="text">
           <string>教师工号：</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_16">
          <property name="text">
           <string>0000000000</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QTabWidget" name="tabWidget">
      <property name="currentIndex">
       <number>1</number>
      </property>
      <widget class="QWidget" name="tab">
       <attribute name="title">
        <string>我的课表</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QTabWidget" name="tabWidget_2">
          <property name="tabPosition">
           <enum>QTabWidget::West</enum>
          </property>
          <property name="currentIndex">
           <number>1</number>
          </property>
          <widget class="QWidget" name="tab_4">
           <attribute name="title">
            <string> 列表显示</string>
           </attribute>
           <layout class="QVBoxLayout" name="verticalLayout_3">
            <item>
             <widget class="QTableWidget" name="teacherCourses">
              <column>
               <property name="text">
                <string>课程编号</string>
               </property>
              </column>
              <column>
               <property name="text">
                <string>课程名称</string>
               </property>
              </column>
              <column>
               <property name="text">
                <string>日期</string>
               </property>
              </column>
              <column>
               <property name="text">
                <string>开始节数</string>
               </property>
              </column>
              <column>
               <property name="text">
                <string>结束节数</string>
               </property>
              </column>
              <column>
               <property name="text">
                <string>上课地点</string>
               </property>
              </column>
             </widget>
            </item>
           </layout>
          </widget>
          <widget class="QWidget" name="tab_5">
           <attribute name="title">
            <string>表格显示</string>
           </attribute>
           <widget class="QTableWidget" name="teacherCourses_2">
            <property name="geometry">
             <rect>
              <x>10</x>
              <y>10</y>
              <width>694</width>
              <height>400</height>
             </rect>
            </property>
            <row>
             <property name="text">
              <string>1</string>
             </property>
            </row>
            <row>
             <property name="text">
              <string>2</string>
             </property>
            </row>
            <row>
             <property name="text">
              <string>3</string>
             </property>
            </row>
            <row>
             <property name="text">
              <string>4</string>
             </property>
            </row>
            <row>
             <property name="text">
              <string>5</string>
             </property>
            </row>
            <row>
             <property name="text">
              <string>6</string>
             </property>
            </row>
            <row>
             <property name="text">
              <string>7</string>
             </property>
            </row>
            <row>
             <property name="text">
              <string>8</string>
             </property>
            </row>
            <row>
             <property name="text">
              <string>9</string>
             </property>
            </row>
            <row>
             <property name="text">
              <string>10</string>
             </property>
            </row>
            <row>
             <property name="text">
              <string>11</string>
             </property>
            </row>
            <row>
             <property name="text">
              <string>12</string>
             </property>
            </row>
            <row>
             <property name="text">
              <string>13</string>
             </property>
            </row>
            <column>
             <property name="text">
              <string>周一</string>
             </property>
            </column>
            <column>
             <property name="text">
              <string>周二</string>
             </property>
            </column>
            <column>
             <property name="text">
              <string>周三</string>
             </property>
            </column>
            <column>
             <property name="text">
              <string>周四</string>
             </property>
            </column>
            <column>
             <property name="text">
              <string>周五</string>
             </property>
            </column>
            <column>
             <property name="text">
              <string>周六</string>
             </property>
            </column>
            <column>
             <property name="text">
              <string>周日</string>
             </property>
            </column>
           </widget>
          </widget>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_3">
       <attribute name="title">
        <string>课程管理</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_4">
        <item>
         <widget class="QTabWidget" name="tabWidget_3">
          <property name="tabPosition">
           <enum>QTabWidget::West</enum>
          </property>
          <property name="currentIndex">
           <number>1</number>
          </property>
          <widget class="QWidget" name="tab_6">
           <attribute name="title">
            <string>选课情况</string>
           </attribute>
           <layout class="QVBoxLayout" name="verticalLayout_7">
            <item>
             <widget class="QTableWidget" name="tableWidget_2">
              <column>
               <property name="text">
                <string>课程编号</string>
               </property>
              </column>
              <column>
               <property name="text">
                <string>课程名</string>
               </property>
              </column>
              <column>
               <property name="text">
                <string>日期</string>
               </property>
              </column>
              <column>
               <property name="text">
                <string>开始节数</string>
               </property>
              </column>
              <column>
               <property name="text">
                <string>结束节数</string>
               </property>
              </column>
              <column>
               <property name="text">
                <string>最大容量</string>
               </property>
              </column>
              <column>
               <property name="text">
                <string>现有容量</string>
               </property>
              </column>
             </widget>
            </item>
           </layout>
          </widget>
          <widget class="QWidget" name="tab_7">
           <attribute name="title">
            <string>名单管理</string>
           </attribute>
           <widget class="QTableWidget" name="tableWidget">
            <property name="geometry">
             <rect>
              <x>9</x>
              <y>49</y>
              <width>691</width>
              <height>341</height>
             </rect>
            </property>
            <column>
             <property name="text">
              <string>学生编号</string>
             </property>
            </column>
            <column>
             <property name="text">
              <string>学生姓名</string>
             </property>
            </column>
            <column>
             <property name="text">
              <string>班级</string>
             </property>
            </column>
            <column>
             <property name="text">
              <string>年级</string>
             </property>
            </column>
            <column>
             <property name="text">
              <string>成绩</string>
             </property>
            </column>
           </widget>
           <widget class="QWidget" name="layoutWidget">
            <property name="geometry">
             <rect>
              <x>30</x>
              <y>390</y>
              <width>671</width>
              <height>25</height>
             </rect>
            </property>
            <layout class="QHBoxLayout" name="horizontalLayout">
             <item>
              <spacer name="horizontalSpacer">
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>588</width>
                 <height>20</height>
                </size>
               </property>
              </spacer>
             </item>
             <item>
              <widget class="QPushButton" name="pushButton">
               <property name="text">
                <string>成绩录入</string>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
           <widget class="QWidget" name="layoutWidget">
            <property name="geometry">
             <rect>
              <x>30</x>
              <y>10</y>
              <width>487</width>
              <height>25</height>
             </rect>
            </property>
            <layout class="QHBoxLayout" name="horizontalLayout_2">
             <item>
              <widget class="QLabel" name="label">
               <property name="text">
                <string>课程编号：</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="lineEdit_3"/>
             </item>
             <item>
              <widget class="QLabel" name="label_2">
               <property name="text">
                <string>课程名称：</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="lineEdit_4"/>
             </item>
             <item>
              <widget class="QPushButton" name="pushButton_2">
               <property name="text">
                <string>搜索</string>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </widget>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_2">
       <attribute name="title">
        <string>个人信息</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_5">
        <item>
         <widget class="QTabWidget" name="tabWidget_4">
          <property name="tabPosition">
           <enum>QTabWidget::West</enum>
          </property>
          <property name="currentIndex">
           <number>0</number>
          </property>
          <widget class="QWidget" name="tab_8">
           <attribute name="title">
            <string>课程信息</string>
           </attribute>
           <widget class="QPlainTextEdit" name="plainTextEdit">
            <property name="geometry">
             <rect>
              <x>9</x>
              <y>93</y>
              <width>694</width>
              <height>124</height>
             </rect>
            </property>
           </widget>
           <widget class="QTableWidget" name="tableWidget_3">
            <property name="geometry">
             <rect>
              <x>9</x>
              <y>254</y>
              <width>694</width>
              <height>124</height>
             </rect>
            </property>
            <column>
             <property name="text">
              <string>课程编号</string>
             </property>
            </column>
            <column>
             <property name="text">
              <string>课程名称</string>
             </property>
            </column>
            <column>
             <property name="text">
              <string>专业</string>
             </property>
            </column>
            <column>
             <property name="text">
              <string>课程介绍</string>
             </property>
             <property name="textAlignment">
              <set>AlignLeading|AlignVCenter</set>
             </property>
            </column>
           </widget>
           <widget class="QWidget" name="layoutWidget">
            <property name="geometry">
             <rect>
              <x>10</x>
              <y>10</y>
              <width>406</width>
              <height>22</height>
             </rect>
            </property>
            <layout class="QHBoxLayout" name="horizontalLayout_3">
             <item>
              <widget class="QLabel" name="label_3">
               <property name="text">
                <string>课程编号：</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="lineEdit"/>
             </item>
             <item>
              <widget class="QLabel" name="label_4">
               <property name="text">
                <string>课程名称：</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="lineEdit_2"/>
             </item>
            </layout>
           </widget>
           <widget class="QWidget" name="layoutWidget">
            <property name="geometry">
             <rect>
              <x>10</x>
              <y>70</y>
              <width>586</width>
              <height>22</height>
             </rect>
            </property>
            <layout class="QHBoxLayout" name="horizontalLayout_9">
             <item>
              <widget class="QLabel" name="label_9">
               <property name="text">
                <string>课程介绍：</string>
               </property>
              </widget>
             </item>
             <item>
              <spacer name="horizontalSpacer_8">
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>518</width>
                 <height>20</height>
                </size>
               </property>
              </spacer>
             </item>
            </layout>
           </widget>
           <widget class="QWidget" name="layoutWidget">
            <property name="geometry">
             <rect>
              <x>100</x>
              <y>220</y>
              <width>598</width>
              <height>25</height>
             </rect>
            </property>
            <layout class="QHBoxLayout" name="horizontalLayout_6">
             <item>
              <spacer name="horizontalSpacer_4">
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>388</width>
                 <height>20</height>
                </size>
               </property>
              </spacer>
             </item>
             <item>
              <widget class="QPushButton" name="pushButton_4">
               <property name="text">
                <string>提出申请</string>
               </property>
              </widget>
             </item>
             <item>
              <spacer name="horizontalSpacer_7">
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>40</width>
                 <height>20</height>
                </size>
               </property>
              </spacer>
             </item>
             <item>
              <widget class="QPushButton" name="pushButton_5">
               <property name="text">
                <string>清空</string>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
           <widget class="QWidget" name="layoutWidget">
            <property name="geometry">
             <rect>
              <x>280</x>
              <y>380</y>
              <width>421</width>
              <height>25</height>
             </rect>
            </property>
            <layout class="QHBoxLayout" name="horizontalLayout_10">
             <item>
              <spacer name="horizontalSpacer_9">
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>338</width>
                 <height>20</height>
                </size>
               </property>
              </spacer>
             </item>
             <item>
              <widget class="QPushButton" name="pushButton_6">
               <property name="text">
                <string>删除课程</string>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
           <widget class="QWidget" name="layoutWidget">
            <property name="geometry">
             <rect>
              <x>10</x>
              <y>40</y>
              <width>495</width>
              <height>22</height>
             </rect>
            </property>
            <layout class="QHBoxLayout" name="horizontalLayout_4">
             <item>
              <widget class="QLabel" name="label_5">
               <property name="text">
                <string>专业：</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="lineEdit_5"/>
             </item>
             <item>
              <spacer name="horizontalSpacer_5">
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>288</width>
                 <height>20</height>
                </size>
               </property>
              </spacer>
             </item>
            </layout>
           </widget>
          </widget>
          <widget class="QWidget" name="tab_9">
           <attribute name="title">
            <string>个人信息</string>
           </attribute>
           <widget class="QWidget" name="layoutWidget">
            <property name="geometry">
             <rect>
              <x>130</x>
              <y>20</y>
              <width>177</width>
              <height>22</height>
             </rect>
            </property>
            <layout class="QHBoxLayout" name="horizontalLayout_23">
             <item>
              <widget class="QLabel" name="label_17">
               <property name="text">
                <string>工号</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="lineEdit_7"/>
             </item>
            </layout>
           </widget>
           <widget class="QPushButton" name="pushButton_8">
            <property name="geometry">
             <rect>
              <x>170</x>
              <y>270</y>
              <width>75</width>
              <height>23</height>
             </rect>
            </property>
            <property name="text">
             <string> 执行修改</string>
            </property>
           </widget>
           <widget class="QWidget" name="layoutWidget_2">
            <property name="geometry">
             <rect>
              <x>130</x>
              <y>100</y>
              <width>177</width>
              <height>22</height>
             </rect>
            </property>
            <layout class="QHBoxLayout" name="horizontalLayout_21">
             <item>
              <widget class="QLabel" name="label_18">
               <property name="text">
                <string>职称：</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="lineEdit_9"/>
             </item>
            </layout>
           </widget>
           <widget class="QWidget" name="layoutWidget_3">
            <property name="geometry">
             <rect>
              <x>130</x>
              <y>200</y>
              <width>177</width>
              <height>22</height>
             </rect>
            </property>
            <layout class="QHBoxLayout" name="horizontalLayout_11">
             <item>
              <widget class="QLabel" name="label_24">
               <property name="text">
                <string>邮箱：</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="lineEdit_12"/>
             </item>
            </layout>
           </widget>
           <widget class="QWidget" name="layoutWidget_4">
            <property name="geometry">
             <rect>
              <x>130</x>
              <y>170</y>
              <width>177</width>
              <height>22</height>
             </rect>
            </property>
            <layout class="QHBoxLayout" name="horizontalLayout_12">
             <item>
              <widget class="QLabel" name="label_23">
               <property name="text">
                <string>电话：</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="lineEdit_11"/>
             </item>
            </layout>
           </widget>
           <widget class="QWidget" name="layoutWidget_5">
            <property name="geometry">
             <rect>
              <x>130</x>
              <y>60</y>
              <width>177</width>
              <height>22</height>
             </rect>
            </property>
            <layout class="QHBoxLayout" name="horizontalLayout_22">
             <item>
              <widget class="QLabel" name="label_19">
               <property name="text">
                <string>姓名：</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="lineEdit_8"/>
             </item>
            </layout>
           </widget>
           <widget class="QWidget" name="layoutWidget_6">
            <property name="geometry">
             <rect>
              <x>130</x>
              <y>140</y>
              <width>177</width>
              <height>22</height>
             </rect>
            </property>
            <layout class="QHBoxLayout" name="horizontalLayout_20">
             <item>
              <widget class="QLabel" name="label_20">
               <property name="text">
                <string>专业：</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="lineEdit_10"/>
             </item>
            </layout>
           </widget>
          </widget>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>779</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
